src/components/event_forms/EventForm.jsx

```diff
@@ -362,7 +362,7 @@ const EventForm = (props) => {
   }
 
   // if user attempt to submit empty form throw error
-  if (Object.keys(formErrors).length && !fileUpload) {
+  if (Object.keys(formErrors).length && !fileUpload && formErrors === 'add') {
```

Comment:
Is this supposed to be `fileType === add` ? I'm not familiar with that value for formErrors

Reply:
wow yes, good catch fixing that now